### ISSUE TITLE
How to auth nsqadmin users?

### ISSUE THREAD

**Issue by jetspeed** [jehiah/nsqauth-contrib#4]
How this project auth nsqadmin users? I found no doc from nsq.io  nor from the README.

I run this two py files in 4181 port, and have set auth_http_address to 4181 port, but no auth required when I want to use nsqadmin .

**Comment by tpherndon** [comment]
Correct, `nsqadmin` does not itself have any authentication. This project, `nsqauth`, does not provide authentication for nsqadmin itself, only for adding an authentication layer on top of nsqlookupd and nsqd, to enable downstream/third-party users to consume specific message streams. If you need to provide an authentication layer in front of nsqadmin, you should run nsqadmin behind a proxy server that handles the authentication. That could be Apache or nginx, using HTTP basic auth, or it could be a specialized authentication proxy.
